spec/redis_spec.cr

```diff
@@ -191,6 +191,42 @@ describe Redis::Client do
     end
   end
 
+  describe "sorted sets" do
+    it "can add and remove members of a sorted set" do
+      key = random_key
+
+      begin
+        redis.zadd(key, "1", "a").should eq 1
+        redis.zadd(key, "1", "a").should eq 0
+        redis.zrange(key, "0", "-1").should eq %w[a]
+
+        redis.zadd key, "2", "b"
+        redis.zrange(key, "0", "-1").as(Array).should contain "a"
+        redis.zrange(key, "0", "-1").as(Array).should contain "b"
+
+        redis.zrem key, "a"
+        redis.zrange(key, "0", "-1").as(Array).should_not contain "a"
+        redis.zrange(key, "0", "-1").as(Array).should contain "b"
+      ensure
+        redis.del key
+      end
+    end
```

Comment:
I recognize you used this test structure because it exists elsewhere in this file, and I appreciate you trying to stick with existing conventions. Unfortunately, those tests still using this structure is an oversight on my part.

If you use the `test` macro at the top of this file, you only need to supply what's in the `begin` block:
```suggestion
    test "can add and remove members of a sorted set" do
      redis.zadd(key, "1", "a").should eq 1
      redis.zadd(key, "1", "a").should eq 0
      redis.zrange(key, "0", "-1").should eq %w[a]

      redis.zadd key, "2", "b"
      redis.zrange(key, "0", "-1").as(Array).should contain "a"
      redis.zrange(key, "0", "-1").as(Array).should contain "b"

      redis.zrem key, "a"
      redis.zrange(key, "0", "-1").as(Array).should_not contain "a"
      redis.zrange(key, "0", "-1").as(Array).should contain "b"
    end
```